galasa-ui/src/components/test-runs/OverviewTab.tsx

```diff
@@ -10,10 +10,19 @@ import InlineText from "./InlineText";
 import { Tag } from "@carbon/react";
 import { RunMetadata } from "@/utils/interfaces";
 import { useTranslations } from "next-intl";
+import { Link } from "@carbon/react";
+import { Launch } from "@carbon/icons-react";
+import { getOneMonthAgo, getTwoDaysAgo } from "@/utils/timeOperations";
 
 const OverviewTab = ({ metadata }: { metadata: RunMetadata }) => {
   const tags = metadata?.tags || [];
   const translations = useTranslations("OverviewTab");
+  
+  const TWO_DAYS_GO = getTwoDaysAgo();
+  const MONTH_AGO = getOneMonthAgo();
+  const COMMON_LINK_URL = `/test-runs?runName=${metadata?.runName}&bundle=${metadata?.bundle}&group=${metadata?.group}&package=${metadata?.package}`;
+  const SAME_TEST_RUN_LINK = `${COMMON_LINK_URL}&from=${MONTH_AGO}`;
+  const OTHER_ATTEMPTS_AT_TEST = COMMON_LINK_URL + `&submissionId=${metadata?.submissionId}&from=${TWO_DAYS_GO}`;
```

Comment:
Does `OTHER_ATTEMPTS_AT_TEST` refer to getting the re-queues of the same test (e.g. RUN1-2, RUN1-3, etc.)? Or does it refer to running the same test as a different submission (e.g. a different `galasactl runs submit` command)?

If it means running the same test as a different submission, those runs may not have the same group (if no group is given in the `galasactl runs submit`, it'll get an auto-generated group ID) and will not have the same submission ID as the original test. This means that the link won't return the expected results.

Might need to clarify the name of the constant to avoid this ambiguity.